src/sass/app_2/_common/components/label.scss

```diff
@@ -0,0 +1,34 @@
+.label {
+    font-size: 14px;
+    font-weight: bold;
+    height: 24px;
+    line-height: 20px;
+    border-radius: $BORDER_RADIUS * 3;
```

Comment:
`* 3`?

Reply:
Yes, in case due to design choice we change $BORDER_RADIUS to 0, label won't be rounded.